app/code/Magento/Catalog/Controller/Adminhtml/Product/SuggestAttributeSets.php

```diff
@@ -6,7 +6,7 @@
  */
 namespace Magento\Catalog\Controller\Adminhtml\Product;
 
-class SuggestAttributeSets extends \Magento\Backend\App\Action
+class SuggestAttributeSets extends \Magento\Catalog\Controller\Adminhtml\Product
```

Comment:
I thought that but I was confused by logic in SuggestAttributes.php. Now I've checked how it works and refactored both files.